### ISSUE TITLE
Delete all scraped scenes not currently downloaded

### ISSUE THREAD

**Issue by beamblitz** [xbapps/xbvr#1704]
I have something like 15 gigabytes in xbvr.log. I have something like 20k scenes' worth of metadata downloaded but only ~150 scenes. A way to delete all scraped data *except* for downloaded scenes would be great.

**Comment by theRealKLH** [comment]
and...... then it would just rescrape all that metadata.

**Comment by beamblitz** [comment]
@theRealKLH not necessarily true. You could just turn off the scrapers and scrape new scenes individually. That might not be sustainable/scaleable, but after seeing the eye-popping amount of space the metadata gobbles up, I would prefer to individually scrape any scene I grab here or there moving forward.

**Comment by theRealKLH** [comment]
That is true. But I leave all "active" scrapers I'm slightly interested in enabled. Got no time to be scraping single scenes from 2018. It kinda defeats the purpose of the utility. Scrapes are to have the data BEFORE you know you want the video.. Grin 

Just thinking... You mentioned 15gb in xbvr.log. If this is what you are equating to your metadata size, it's not your metadata.. Just delete the log file. Boom... instant 15gb recovered. 

What is the size of your main.db (metadata)? can't be THAT big.. maybe you are talking preview videos directory (video_preview) or images/covers directory (imageproxy)?

**Comment by beamblitz** [comment]
ah, okay, thanks for explaining that. what is contained in the log file, out of curiosity? (i have some technical skills, but i'm not an engineer, so the thought of peeking in a file that big using the methods i'm aware of gives me hives.) nothing i'll need to redownload?

my main.db is ~150mb, which is definitely not big enough for me to worry about.

and just fwiw, my suggestion was more an add'l feature, to include the option to delete _unused_ scraped data, not to, uh, "scrap" scrapers altogether 😅. but if we're talking about a file that isn't even a gb in my case, that seems hardly worth the effort.

**Comment by theRealKLH** [comment]
> ah, okay, thanks for explaining that. what is contained in the log file, out of curiosity? (i have some technical skills, but i'm not an engineer, so the thought of peeking in a file that big using the methods i'm aware of gives me hives.) nothing i'll need to redownload?
> 

The log file just shows the activity happening (pretty much EVERYTHING is logged). it's not needed unless you have a problem which the error may be captured OR you just like looking at log files...
![image](https://github.com/xbapps/xbvr/assets/65736720/07cab0b4-4f10-4e91-bb7c-a16e50f17899)


> my main.db is ~150mb, which is definitely not big enough for me to worry about.
> 
> and just fwiw, my suggestion was more an add'l feature, to include the option to delete _unused_ scraped data, not to, uh, "scrap" scrapers altogether 😅. but if we're talking about a file that isn't even a gb in my case, that seems hardly worth the effort.
> 
Didn't think you meant to get rid of the scrapers. But if you delete "all unused scraped" data, the ONLY way you have to find out if you MAY want a scene is to go to the web site and search around. How about those times where you just watched a scene with "Betty Boop" from ToonsGoingWildVR.com and now you want to know what else is out there that she starred in? WELL... you deleted all that wonderful metadata from ToonsGoingWildVR (so you would have to visit the site unless you already downloaded all the videos with her), you also deleted ToonsDoingTheDirtyVR.com metadata, NastyToonsVR.com metadata, and so on.. So you have no clue what's available, ONLY the scene that you just watched.
 
> edit: nvm, i moved the file and launched XBVR, then opened the new log file it created. it's 470 bytes and literally just a log. how tf does that get to be 15gb? am i that much of a perv?
Answering as I'm reading so just finding out you already "deleted" the file. Big size doesn't equal perv (So they keep telling me!)
